### PR TITLE
Add docs for supressing sniffs

### PR DESCRIPTION
**Pull request by ibrahim-mubarak**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| Fixed tickets | #267 

Add simple instructions to suppress sniffs

### CODE REVIEW

**Comment by ibrahim-mubarak** [comment]
I was wondering if we should take this opportunity to close #105 and #264

File: docs/insights/README.md [hunk 1]
```diff
@@ -119,3 +119,22 @@ For example, to remove "Unused Parameters" Insight only for some file:
...
+     *
+     * @return bool
+     */
+    public function viewAny(?User $user)
+    {
+        return true;
+    }
+```
+
+You can see [slevomat/coding-standard docs](https://github.com/slevomat/coding-standard#suppressing-sniffs-locally) for more info
```
Comment (nunomaduro) [inline docs/insights/README.md]: This is a feature specific to `slevomat` insights?

Comment (olivernybroe) [reply docs/insights/README.md]: I think that it is specific for Slevomat, as it has to know the context to only silence it for that code block.

In phpcs you have the `phpcs:disable` and `phpcs:enable` commands to specify which code blocks you want to ignore, however Slevomat is using the context of the sniff instead to figure out which part you want to ignore. The `phpcs:disable` command also supports specifying a sniff.

**Comment by JustSteveKing** [comment]
Closing as stale PR

**Pull request closed without merging**